common/views/components/ImageGalleryV2/ImageGalleryV2.js

```diff
@@ -131,6 +135,37 @@ class ImageGallery extends Component<Props, State> {
                     </Fragment>
                   )}
 
+                  <div
+                    style={{
+                      position: 'sticky',
+                      top: '10px',
+                      paddingRight: '10px',
+                      paddingBottom: '10px',
+                      display: 'flex',
+                      justifyContent: 'flex-end',
+                      visibility: isActive ? 'visible' : 'hidden',
+                      zIndex: 3,
```

Comment:
@wellcometrust/experience-devs 

When do we not want to be doing this?
I know it's a test, but we have classes for almost all of the above in `_root`?
There's a bigger question I think about how we use `_root` with `styled-components`?

Reply:
Was wondering the same (about if/how to share styled-components and root-scope-classes).

Would be nice to have this using styled components, though.

Have dropped a `TODO` in there.

Reply:
Let's have a chat about this while chewing the code.